app/views/import/fogbugz/new.html.haml

```diff
@@ -0,0 +1,45 @@
+- page_title "Fogbugz Import"
+%h3.page-title
+  %i.fa.fa-bug
+  Import projects from Fogbugz
+%hr
+
+= form_tag callback_import_fogbugz_path, class: 'form-horizontal', multipart: true do
+  %p
+    Follow the steps below to export your Fogbugz project data.
+    In the next step, you'll be able to select the projects you want to import.
+  %ol
+    %li
+      %p
+        Log into your Fogbugz account and set your filter to show All cases.
+      %p
+        =image_tag 'fogbugz_filter.png'
+    %li
+      %p
+        Enter your Fogbugz URL and login information.
+      %p
+        = label_tag :uri do
+          Fogbugz URL
+          = text_field_tag :uri
```

Comment:
Could you add a `placeholder` with an example? Right now, it isn't clear if this should be just the host or a full path.